sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -52,6 +54,8 @@ public void start() {
           addWarn("Failed to init Sentry during appender initialization: " + e.getMessage());
         }
       } else {
+        // NOTE: logback.xml properties will not be applied in this case as the SDK has already been
+        // initialized
```

Comment:
I'd rather improve the logging message below instead of adding a comment in the code.

Reply:
Hmm looks like I got put this in the wrong `else` path, will update.

Reply:
Updated